fix having inapropriate ammo in CL lockbox.

Huffie56 opened the pull request:
# About the pull request
fixes: https://github.com/cmss13-devs/cmss13/issues/3976
<!-- Remove this text and explain what the purpose of your PR is.

Mention if you have tested your changes. If you changed a map, make sure you used the mapmerge tool.
If this is an Issue Correction, you can type "Fixes Issue #169420" to link the PR to the corresponding Issue number #169420.

Remember: something that is self-evident to you might not be to others. Explain your rationale fully, even if you feel it goes without saying. -->

# Explain why it's good for the game
removing the improper ammo in the lock box because it's obsolete considering the new gun CL as.
adding a way to buy more ammo so that CL as a way to refill is weapon.
didn't wanted to give it for free in the lockbox.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

![image](https://github.com/cmss13-devs/cmss13/assets/117036822/f9aeb4e9-e11f-4a3f-9ccd-da244ee74713)

![image](https://github.com/cmss13-devs/cmss13/assets/117036822/815b6f6a-0532-4beb-9a68-7c524ead3eb1)


Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>


# Changelog
:cl:
balance: adding the possibility to buy "ES-4 stun magazine" for 10 point in the portable vendor.
fix: remove a bunch of ammo that where improper for ES-4 gun that are in the lockbox.
/:cl:

Zonespace27 reviewed (changes requested):
Your PR doesn't explain *why* the CL should lose their roundstart ammo

MusselMain commented:
> Your PR doesn't explain _why_ the CL should lose their roundstart ammo

as i mentioned in my bug report: the CL no longer spawns with the 88 Mod4 in their safe, instead spawning with the ES-4 electrostatic pistol, despite this the CL still receives a lockbox full of 88Mod4 rubber and AP ammo, when I asked a mentor where the CL could find a 88 mod4 to use this ammo, they responded with asking a marine to vend them one from a marine prep room, based on this I assume it's an oversight for the CL to have ammo for a weapon they aren't issued, and that they'd have to ask others for. The new sidearm the CL is issued only gets 3 magazines and it seems like there is no other place to find more, this PR not only solves the confusing question of "why would WY issue a nonlethal weapon to the CL, yet give them lethal ammo for a weapon they aren't issued", but also solves the problem of ES-4 ammo being extremely rare.

Zonespace27 commented:
> <snip>

No, i'm looking for justification for nuking the lockbox ammo entirely, as this PR does

Huffie56 commented:
> > <snip>
> 
> No, i'm looking for justification for nuking the lockbox ammo entirely, as this PR does

i removed /obj/item/storage/lockbox/loyalty/fill_preset_inventory() thinking it would work without it.
just fixed it i added screenshoot in testing if you want to see.
sorry for the confusion i forgot github doesn't test everything...

Zonespace27 commented:
> \<snip\>

This still fails to address the issue I raised of a pack of justification.

Huffie56 commented:
> > <snip>
> 
> This still fails to address the issue I raised of a pack of justification.

? i am confused i am not removing the lock-box just the ammo in it because it's obsolete considering the new gun CL as.
i don't see any reason to keep random magazine including lethal ammo in a lock-box.
Because if CL need to use them he could just get gun and ammo at requisition.

MusselMain commented:
> > <snip>
> 
> No, i'm looking for justification for nuking the lockbox ammo entirely, as this PR does

actually. fair point, having an empty lockbox seems a bit silly.

MusselMain commented on code/game/objects/items/storage/lockbox.dm:
```diff
@@ -54,11 +54,6 @@
 	req_access = list(ACCESS_WY_CORPORATE)
 
 /obj/item/storage/lockbox/loyalty/fill_preset_inventory()
-	new /obj/item/ammo_magazine/pistol/mod88(src)
-	new /obj/item/ammo_magazine/pistol/mod88(src)
-	new /obj/item/ammo_magazine/pistol/mod88/rubber(src)
-	new /obj/item/ammo_magazine/pistol/mod88/rubber(src)
-
```
> Don't leave this empty else you're just giving an empty box with a lock on it. No issues with giving 2-3 spare mags for ES4 in here. Alternatively, move the ES4 from the CL's safe and put it in here with its current ammo cap.

If the sidearm and ammo is moved to the lockbox, that shouldn't be an issue outside of the closet being easier to break into then the safe

Huffie56 replied:
i am just going to add 2 spare mags for the ES4
don't want to do any mapping on almayer right now to much PR causing merge conflicts....
